backend/src/main/resources/application-dev.properties

```diff
@@ -1 +1,14 @@
-brevity.frontend.url=http://localhost:3000
```

Comment:
how does application-dev work? Im guessing its so we can have seperate settings for heroku but how does it know when to apply the application-dev properties?

Reply:
I have in the run script where I set `spring.actives.profile` to dev and on heroku I set it to prod